### PR TITLE
Symbols generation for common libraries

### PR DESCRIPTION
**Pull request by agr**
Currently, sign build queue expects symbols packages to be present, so build script is updated to generate those for common libraries.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -158,12 +162,12 @@ Invoke-BuildStep 'Creating artifacts' {
...
             "src/Validation.PackageSigning.ProcessSignature/Validation.PackageSigning.ProcessSignature.csproj", `
-            "src/Validation.PackageSigning.ValidateCertificate/Validation.PackageSigning.ValidateCertificate.csproj", `
-            "src/NuGet.Jobs.Common/NuGet.Jobs.Common.csproj", `
-            "src/Validation.Common.Job/Validation.Common.Job.csproj"
+            "src/Validation.PackageSigning.ValidateCertificate/Validation.PackageSigning.ValidateCertificate.csproj" `
+            + $CommonLibsProjects
 
         Foreach ($Project in $Projects) {
-            New-Package (Join-Path $PSScriptRoot "$Project") -Configuration $Configuration -BuildNumber $BuildNumber -Version $SemanticVersion -Branch $Branch -MSBuildVersion "$msBuildVersion"
+        	$Symbols = $CommonLibsProjects -contains $Project;
```
Comment (loic-sharma) [inline build.ps1]: There's a tab on this line

Comment (loic-sharma) [reply build.ps1]: Why don't we build symbols for all projects?

Comment (agr) [reply build.ps1]: not anymore

Comment (agr) [reply build.ps1]: Regarding symbols: they are only needed for myget publishing. We didn't publish stuff from this repo on Myget, so no symbols needed. If you are asking why not just create symbols for everything: I don't want to create more mess in artifacts than we need.

File: build.ps1 [hunk 2]
```diff
@@ -158,12 +165,12 @@ Invoke-BuildStep 'Creating artifacts' {
             "src/NuGet.Services.Validation.Orchestrator/NuGet.Services.Validation.Orchestrator.csproj", `
             "src/Stats.CollectAzureChinaCDNLogs/Stats.CollectAzureChinaCDNLogs.csproj", `
             "src/Validation.PackageSigning.ProcessSignature/Validation.PackageSigning.ProcessSignature.csproj", `
-            "src/Validation.PackageSigning.ValidateCertificate/Validation.PackageSigning.ValidateCertificate.csproj", `
-            "src/NuGet.Jobs.Common/NuGet.Jobs.Common.csproj", `
-            "src/Validation.Common.Job/Validation.Common.Job.csproj"
+            "src/Validation.PackageSigning.ValidateCertificate/Validation.PackageSigning.ValidateCertificate.csproj" `
+            + $CommonLibsProjects
```
Comment (loic-sharma) [inline build.ps1]: `CommonLibsProjects` -> `ProjectsWithSymbols`

**Pull request merged**